### ISSUE TITLE
Alt-Enter menu for files and folders is blank

### ISSUE THREAD

**Issue by tendstofortytwo** [Ulauncher/Ulauncher#1209]
### Environment

Fedora Workstation 37

### Bug description

When I type `~/Desk` and then press Alt-Enter to get the alternate menu, Ulauncher instead shows nothing. This is Ulauncher 5.15.1 from the Fedora official repositories. I read through the 6.0 changelog but couldn't find anything that sounded like this, not sure if I missed something though.

![image](https://user-images.githubusercontent.com/5107795/221970981-ca098561-c8d0-4053-b1b8-2e15c3eecb46.png)

### Log output

```shell
2023-02-28 15:30:16,461 | [1;37mINFO[0m | ulauncher: main() | Ulauncher version 0.0.0
2023-02-28 15:30:16,461 | [1;37mINFO[0m | ulauncher: main() | Extension API version 2.0.0
2023-02-28 15:30:16,461 | [1;37mINFO[0m | ulauncher: main() | GTK+ 3.24.36
2023-02-28 15:30:16,461 | [1;37mINFO[0m | ulauncher: main() | Is Wayland: False
2023-02-28 15:30:16,461 | [1;37mINFO[0m | ulauncher: main() | Wayland compatibility: off
2023-02-28 15:30:16,468 | [1;34mDEBUG[0m | ulauncher.ui.windows.Builder: __init__() | consider using a pythonic name instead of design name 'input-box'
2023-02-28 15:30:16,471 | [1;34mDEBUG[0m | ulauncher.ui.windows.Builder: do_connect() | connect builder by name 'ulauncher_window','destroy', 'on_destroy'
2023-02-28 15:30:16,471 | [1;34mDEBUG[0m | ulauncher.ui.windows.Builder: do_connect() | connect builder by name 'ulauncher_window','focus_in_event', 'on_focus_in_event'
2023-02-28 15:30:16,471 | [1;34mDEBUG[0m | ulauncher.ui.windows.Builder: do_connect() | connect builder by name 'ulauncher_window','focus_out_event', 'on_focus_out_event'
2023-02-28 15:30:16,472 | [1;34mDEBUG[0m | ulauncher.ui.windows.Builder: do_connect() | connect builder by name 'input','key_press_event', 'on_input_key_press_event'
2023-02-28 15:30:16,472 | [1;34mDEBUG[0m | ulauncher.ui.windows.Builder: log_unconnected_functions() | Not connected to builder 'on_input_changed'
2023-02-28 15:30:16,473 | [1;34mDEBUG[0m | ulauncher.ui.windows.Builder: log_unconnected_functions() | Not connected to builder 'on_mnu_about_activate'
2023-02-28 15:30:16,473 | [1;34mDEBUG[0m | ulauncher.ui.windows.Builder: log_unconnected_functions() | Not connected to builder 'on_mnu_close_activate'
2023-02-28 15:30:16,473 | [1;34mDEBUG[0m | ulauncher.ui.windows.Builder: log_unconnected_functions() | Not connected to builder 'on_mnu_preferences_activate'
2023-02-28 15:30:16,473 | [1;34mDEBUG[0m | ulauncher.ui.windows.Builder: log_unconnected_functions() | Not connected to builder 'on_preferences_dialog_destroyed'
2023-02-28 15:30:16,474 | [1;34mDEBUG[0m | ulauncher.utils.display: get_screens() | Found 1 monitor(s)
2023-02-28 15:30:16,475 | [1;34mDEBUG[0m | ulauncher.utils.display: get_screens() |   Monitor 0 - X: 0, Y: 0, W: 1920, H: 1080
2023-02-28 15:30:16,482 | [1;34mDEBUG[0m | ulauncher.ui.windows.UlauncherWindow: show_results() | render 0 results
2023-02-28 15:30:16,484 | [1;37mINFO[0m | ulauncher.api.server.ExtensionServer: _start_thread() | Starting WS server on port 5054
2023-02-28 15:30:16,484 | [1;37mINFO[0m | ulauncher.search.apps.app_watcher: start() | Started scanning desktop dirs
2023-02-28 15:30:16,512 | [1;37mINFO[0m | ulauncher.utils.desktop.reader: find_apps_cached() | Found 114 apps in cache
2023-02-28 15:30:16,518 | [1;37mINFO[0m | ulauncher.ui.windows.UlauncherWindow: bind_show_app_hotkey() | Trying to bind app hotkey: <Alt>space
2023-02-28 15:30:16,520 | [1;37mINFO[0m | ulauncher.utils.desktop.reader: read_desktop_file() | Could not read "/home/nsood/.local/share/applications/webex.desktop": constructor returned NULL
2023-02-28 15:30:16,547 | [1;37mINFO[0m | ulauncher.utils.desktop.reader: read_desktop_file() | Could not read "/usr/share/applications/remmina-gnome.desktop": constructor returned NULL
2023-02-28 15:30:16,548 | [1;37mINFO[0m | ulauncher.utils.desktop.reader: read_desktop_file() | Could not read "/usr/share/applications/kcm_trash.desktop": constructor returned NULL
2023-02-28 15:30:16,553 | [1;37mINFO[0m | ulauncher.utils.desktop.reader: read_desktop_file() | Could not read "/usr/share/applications/evolution-calendar.desktop": constructor returned NULL
2023-02-28 15:30:16,558 | [1;37mINFO[0m | ulauncher.utils.desktop.reader: find_apps_cached() | Found 114 apps in the system
2023-02-28 15:30:16,558 | [1;37mINFO[0m | ulauncher.search.apps.app_watcher: start() | Scanned desktop dirs in 0.07 seconds
2023-02-28 15:30:16,563 | [1;34mDEBUG[0m | ulauncher.search.apps.app_watcher: start() | Start watching desktop files...

(ulauncher:278908): Gdk-CRITICAL **: 15:30:17.641: gdk_window_thaw_toplevel_updates: assertion 'window->update_and_descendants_freeze_count > 0' failed
2023-02-28 15:30:18,751 | [1;34mDEBUG[0m | ulauncher.ui.windows.UlauncherWindow: show_results() | render 17 results
2023-02-28 15:30:18,913 | [1;34mDEBUG[0m | ulauncher.ui.windows.UlauncherWindow: show_results() | render 17 results
2023-02-28 15:30:19,454 | [1;34mDEBUG[0m | ulauncher.ui.windows.UlauncherWindow: show_results() | render 0 results
2023-02-28 15:30:19,640 | [1;37mINFO[0m | ulauncher.utils.image_loader: get_app_icon_pixbuf() | Could not load icon for lutris_need-for-speed-underground-2. E: gtk-icon-theme-error-quark: Icon 'lutris_need-for-speed-underground-2' not present in theme Adwaita (0)
2023-02-28 15:30:19,753 | [1;37mINFO[0m | ulauncher.utils.image_loader: get_app_icon_pixbuf() | Could not load icon for preferences-desktop-locale. E: gtk-icon-theme-error-quark: Icon 'preferences-desktop-locale' not present in theme Adwaita (0)
2023-02-28 15:30:19,754 | [1;37mINFO[0m | ulauncher.utils.image_loader: get_app_icon_pixbuf() | Could not load icon for rofi. E: gtk-icon-theme-error-quark: Icon 'rofi' not present in theme Adwaita (0)
2023-02-28 15:30:19,754 | [1;37mINFO[0m | ulauncher.utils.image_loader: get_app_icon_pixbuf() | Could not load icon for rofi. E: gtk-icon-theme-error-quark: Icon 'rofi' not present in theme Adwaita (0)
2023-02-28 15:30:19,824 | [1;34mDEBUG[0m | ulauncher.ui.windows.UlauncherWindow: show_results() | render 7 results
2023-02-28 15:30:20,268 | [1;34mDEBUG[0m | ulauncher.ui.windows.UlauncherWindow: show_results() | render 0 results
2023-02-28 15:30:20,659 | [1;34mDEBUG[0m | ulauncher.ui.windows.UlauncherWindow: show_results() | render 17 results
2023-02-28 15:30:21,189 | [1;34mDEBUG[0m | ulauncher.ui.windows.UlauncherWindow: show_results() | render 17 results
2023-02-28 15:30:21,415 | [1;34mDEBUG[0m | ulauncher.ui.windows.UlauncherWindow: show_results() | render 17 results
2023-02-28 15:30:21,428 | [1;34mDEBUG[0m | ulauncher.ui.windows.UlauncherWindow: show_results() | render 17 results
2023-02-28 15:30:21,495 | [1;34mDEBUG[0m | ulauncher.ui.windows.UlauncherWindow: show_results() | render 17 results
2023-02-28 15:30:22,810 | [1;31mERROR[0m | ulauncher: except_hook() | Uncaught exception
Traceback (most recent call last):
  File "/usr/lib/python3.11/site-packages/ulauncher/ui/windows/UlauncherWindow.py", line 372, in show_results
    results = self.create_item_widgets(result_items, self._get_user_query())
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/ulauncher/ui/windows/UlauncherWindow.py", line 410, in create_item_widgets
    item_frame.initialize(builder, result_item, index, query)
  File "/usr/lib/python3.11/site-packages/ulauncher/ui/ResultItemWidget.py", line 38, in initialize
    self.set_icon(item_object.get_icon())
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/ulauncher/search/file_browser/alt_menu/OpenFolderItem.py", line 27, in get_icon
    return get_themed_icon_by_name('system-file-manager', self.get_icon_size())
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/ulauncher/utils/image_loader.py", line 69, in get_themed_icon_by_name
    return icon_theme.load_icon(icon_name, icon_size, Gtk.IconLookupFlags.FORCE_SIZE)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
gi.repository.GLib.GError: gtk-icon-theme-error-quark: Icon 'system-file-manager' not present in theme Adwaita (0)
2023-02-28 15:30:25,820 | [1;33mWARNING[0m | ulauncher: main() | On KeyboardInterrupt
```

### Communication guidelines

- [X] I have read and agree with Ulauncher's communication guidelines and checked the troubleshooting page for self-help.

### Not an extension issue

- [X] This issue applies to the Ulauncher application, not a specific extension.

### Ulauncher v6

- [X] I have made sure the bug isn't already fixed in v6.

**Comment by friday** [comment]
It happens because [the Gnome/Adwaita maintainers decided it was a great idea to delete standard xdg icon themes](https://gitlab.gnome.org/GNOME/adwaita-icon-theme/-/issues/166),  and not tell anyone, but "see what falls out and then fixing it everywhere" (talking about their own code, not community/app developers).

So you can fix this issue on your end by switching to a standards complaint icon theme or a fallback theme such as hicolor (this one isn't pretty though).

But please keep the issue open. Ulauncher functionality of course shouldn't break because of a missing icon, it should just show a fallback. That's been fixed already in v6. Just not ported to v5.

**Comment by tendstofortytwo** [comment]
ahh, okay. switching to another icon theme does fix this for now, thank you!

**Comment by friday** [comment]
I fixed it, but can't release because Travis CI is very broken.

**Comment by friday** [comment]
Fixed in v5.15.2 that was just released.
